src/app/dock_widgets/process/gt_processdock.cpp

```diff
@@ -919,10 +913,9 @@ GtProcessDock::makeAddMenu(QMenu& menu)
                         gtDataModel->objectFromIndex(srcIdx) : nullptr;
 
     // add empty task action
-    auto addEmptyTask = [this](GtObject* obj){
-        obj ? this->addEmptyTask(obj) : addEmptyTaskToRoot();
-    };
-    auto addemptytask = gt::gui::makeAction(tr("Empty Task"), addEmptyTask)
+    auto addEmptyTask = std::bind(&GtProcessDock::addEmptyTaskToRoot, this);
+    auto addemptytask = gt::gui::makeAction(tr("Empty Task as Root"),
```

Comment:
I would prefer "Empty Root Task" instead. Rest looks fine to me! Good job

```suggestion
    auto addemptytask = gt::gui::makeAction(tr("Empty Root Task"),
```